sql/src/main/java/io/crate/metadata/blob/InternalBlobTableInfoFactory.java

```diff
@@ -96,7 +96,7 @@ public BlobTableInfo create(TableIdent ident, ClusterService clusterService) {
             DocIndexMetaData.getRoutingHashFunction(mappingMap),
             DocIndexMetaData.getVersionCreated(mappingMap),
             DocIndexMetaData.getVersionUpgraded(mappingMap),
-            DocIndexMetaData.isClosed(indexMetaData));
+            DocIndexMetaData.isClosed(indexMetaData, mappingMap, false));
```

Comment:
afaik we can just set it to indexMetaData.state as a blob table can never be partitioned